README.md

```diff
@@ -1,2 +1,10 @@
 # api
-Contains the API definitions used by OLM and Marketplace
+
+Contains the API definitions used by [Operator Lifecycle Manager][olm] (OLM) and [Marketplace Operator][marketplace]
+
+## `pkg/validation`: Operator Manifest Validation
+
+`pkg/validation` exposes a convenient set of interfaces to validate Kubernetes object manifests, primarily for use in an Operator project.
```

Comment:
What do you think of adding some examples? This could be in a separate PR/folder

Reply:
I'm thinking examples of implementing validators / using the apis

Reply:
This might make a nice follow-up.